xds/src/main/java/io/grpc/xds/XdsClient.java

```diff
@@ -81,9 +184,67 @@ ConfigUpdate build() {
    * configurations for traffic control such as drop overloads, inter-cluster load balancing
    * policy and etc.
    */
-  // TODO(zdapeng): content TBD.
   static final class EndpointUpdate {
+    private final String clusterName;
```

Comment:
`edsServiceName` to be more specific?

Reply:
`cluster` in envoy == `service` in google, but we'd better be consistent using `cluster`. I don't think prefixing with `eds` is a good idea here as we do not need to expose the detail that this update comes from EDS.

Reply:
We only talk about envoy since we are in OSS, and service_name for EDS is also an envoy term. But it's fine to use `clusterName` because it's from ClusterLoadAssignment.